validator/sawtooth_validator/journal/back_pressure_handlers.py

```diff
@@ -31,11 +34,21 @@ class ClientBatchSubmitBackpressureHandler(Handler):
     able.  Otherwise it returns a QUEUE_FULL response.
     """
 
-    def __init__(self, can_accept_fn, queue_info_fn):
+    def __init__(self, can_accept_fn, queue_info_fn, metrics_registry=None):
         self._can_accept = can_accept_fn
         self._queue_info = queue_info_fn
         self._applying_backpressure = False
 
+        if metrics_registry:
+            self._bp_batches_rejected_count = CounterWrapper(
+                metrics_registry.counter('bp_batches_rejected_count'))
```

Comment:
The abbreviation is problematic here, as no one knows what bp means.  Does the longer string fail to work?

Reply:
It just required an ugly line break. I'll change it as the variable names are more important in this case.

Reply:
Also fails to pass lint as `backpressure_batches_rejected_count` is over 30 characters. Is `batches_rejected_count` too vague in this context?